### ISSUE TITLE
[Bug]: When an album or singer has more than 200 songs, you cannot add the excess songs to the created playlist no matter how you add them!!!

### ISSUE THREAD

**Issue by Whys8182** [navidrome/navidrome#3320]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [X] I'm using the latest version (your issue may have been fixed already)

### Version

0.53.2

### Current Behavior

When an album or singer has more than 200 songs, you cannot add the excess songs to the created playlist no matter how you add them!!!

### Expected Behavior

When an album or singer has more than 200 songs, you cannot add the excess songs to the created playlist no matter how you add them!!!

### Steps To Reproduce

When an album or singer has more than 200 songs, you cannot add the excess songs to the created playlist no matter how you add them!!!

### Environment

```markdown
- OS:
- Browser:Chrome 110.0.5481.177 on Windows 10
- Client:
```

### How Navidrome is installed?

Docker

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by deluan** [comment]
This is a limitation of the current UI for certain playlist operations, as too much data can make the UI unresponsive.

A workaround is to drag and drop what you want to add to the playlist, like in this:
 

https://github.com/user-attachments/assets/afe4950c-ea4d-48e5-a21e-221189e6562a

I'm not planning to change this in the current UI, but will consider this in the next UI (that should be worked on in the next months)

**Comment by Szeraax** [comment]
How do you want to track this for the future, @deluan?

Github projects? Github discussions? A label? We could add it to a new project in the backlog as wishlist for UI redo pretty easily.

**Comment by deluan** [comment]
Yeah, I think a New UI "project" could be useful to keep track of these requests/issues. Want to take a stab at creating it?

**Comment by Szeraax** [comment]
The way that I do this at work is to have an organized backlog so that when it comes time to do "next ui", I can go and look at issues/FRs that should be considered during planning. So I'm going to start doing the same thing here.
